src/libraries/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -662,8 +655,8 @@ public static IEnumerable<Object[]> Iri_ExpandingContents_AllowedSize
         [MemberData(nameof(Iri_ExpandingContents_AllowedSize))]
         public static void Iri_ExpandingContents_DoesNotThrowIfSizeAllowed(string input)
         {
-            Uri itemUri = new Uri(input);
-            Assert.True(Uri.TryCreate(input, UriKind.Absolute, out Uri itemUri2));
+            _ = new Uri(input);
```

Comment:
For all of these `_ = new Uri(input);`, do we even need the discard?  Can't they just be `new Uri(input);`?

Reply:
I find `_ = new Uri(input);` to be very clear that we are testing that the constructor/getter does not throw, where as just calling the constructor on an empty line feels out of place.

Reply:
For a getter, the discard is required; you can't just invoke a getter and not assign it to anything.  For a constructor, you're obviously invoking something; adding the extra discard and assignment is extra ceremony that IMO just detracts from the actual thing being tested.